usdmanager/utils.py

```diff
@@ -83,21 +83,21 @@ def expandPath(path, parentPath=None, sdf_format_args=None, extractedDir=None):
                 resolver.ConfigureResolverForAsset(path)
             context = resolver.CreateDefaultContextForAsset(path)
             with Ar.ResolverContextBinder(context):
-                anchoredPath = path if parentPath is None else resolver.AnchorRelativePath(parentPath, path)
+                anchoredPath = path if parentPath is None else resolver.CreateIdentifier(path)
```

Comment:
I think we should do an "if" here similar to [this](https://github.com/PixarAnimationStudios/USD/blob/release/pxr/usd/bin/usdresolve/usdresolve.py#L66) to maintain compatibility with older asset resolvers. Thoughts?